pkgs/by-name/am/amazon-ec2-metadata-mock/package.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nix-update-script,
+  versionCheckHook,
+}:
+
+buildGoModule rec {
+  pname = "amazon-ec2-metadata-mock";
+  version = "1.12.0";
+
+  src = fetchFromGitHub {
+    owner = "aws";
+    repo = "amazon-ec2-metadata-mock";
+    rev = "refs/tags/v${version}";
```

```suggestion
    tag = "v${version}";
```
The tag parameter was recently added to make this shorter.

LGTM otherwise.